### PR TITLE
Closes #3292: Move modules out from experimental status

### LINKED ISSUE #3292: Clean up experimental modules
This is a list of all modules in the `The University of Arizona - Experimental` list.

Potential modules to remove from experimental group without any extra work are in bold.

- [az_demo](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_demo/az_demo.info.yml)
- [az_event_trellis](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_event/az_event_trellis/az_event_trellis.info.yml)
- **[az_news_feeds](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_news/az_news_feeds/az_news_feeds.info.yml)**
- [az_publication_import](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_publication/az_publication_import/az_publication_import.info.yml)
- **[az_publication_bibtex](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_publication/az_publication_bibtex/az_publication_bibtex.info.yml)**
- **[az_paragraphs_views_side_by_side](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/az_paragraphs_views_side_by_side/az_paragraphs_views_side_by_side.info.yml)**
- **[az_paragraphs_link_group](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/az_paragraphs_link_group/az_paragraphs_link_group.info.yml)**
- [az_publication_doi](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_publication/az_publication_doi/az_publication_doi.info.yml)
- [az_marketing_cloud](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_marketing_cloud/az_marketing_cloud.info.yml)
- [az_migration](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_migration/az_migration.info.yml)
- **[az_news_export](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_news/az_news_export/az_news_export.info.yml)**
- [az_curated_views](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_curated_views/az_curated_views.info.yml)
- **[az_enterprise_attributes_import](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_core/az_enterprise_attributes_import/az_enterprise_attributes_import.info.yml)**
- **[az_course](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_course/az_course.info.yml)**
- **[az_digital_asset_library](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_digital_asset_library/az_digital_asset_library.info.yml)**
- [az_publication](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_publication/az_publication.info.yml)

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description
Moves modules out from experimental status

- [x] Do I need a DB update? - Thinking not

## Related issues
Closes #3292 

## How to test
1. Open module list
2. See that listed modules are no longer in experimental group

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [x] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_course/az_course.info.yml
Comment (camikazegreen) [inline modules/custom/az_course/az_course.info.yml:1]: I don't think this is ready to be marked stable.

**Comment by tadean** [comment]
Some things to possibly consider:

- A major change to `az_news_export` is potentially coming in https://github.com/az-digital/az_quickstart/pull/3237
- I think `az_publication_bibtex` being in `az_publication` which is experimental might preclude the child module being marked as stable.

File: modules/custom/az_digital_asset_library/az_digital_asset_library.info.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: 'Quickstart Digital Asset Library'
 type: module
 description: 'The connector for the centralized University of Arizona Digital Asset Library (IntelligenceBank DAM).'
 core_version_requirement: ^9 || ^10
-package: 'The University of Arizona - Experimental'
+package: 'The University of Arizona'
```
Comment (danahertzberg) [inline modules/custom/az_digital_asset_library/az_digital_asset_library.info.yml:5]: Look into issues on quickstart.arizona.edu with this module

**Comment by danahertzberg** [comment]
Need to add documentation to RELEASES.md for converting modules from experimental to stable.
- New features without bugs for period of time
- Features that are not expecting any further changes

File: modules/custom/az_digital_asset_library/az_digital_asset_library.info.yml (hunk 1 above)
Comment (danahertzberg) [reply modules/custom/az_digital_asset_library/az_digital_asset_library.info.yml:5]: I don't recall what the issue was and I was able to embed a DAL photo just fine on a draft QS page. @camikazegreen Do you have any more specifics I should look into?

Comment (camikazegreen) [reply modules/custom/az_digital_asset_library/az_digital_asset_library.info.yml:5]: The issue was related to the login screen not using the included config to pre-fill with the appropriate settings to login. 
I'm also ok with considering this to be a site specific issue that does not need to be resolved to move this module to stable. I haven't seen it happen to any other sites.

**Comment by camikazegreen** [comment]
> Need to add documentation to RELEASES.md for converting modules from experimental to stable.
> 
>     * New features without bugs for period of time
> 
>     * Features that are not expecting any further changes
> 
> 
> @camikazegreen to make new issue and PR for this update.

This PR is ready for review: https://github.com/az-digital/az_quickstart/pull/3477

**Comment by danahertzberg** [comment]
Closing in favor of having module-specific changes. See linked issue for example.

**Pull request closed without merging**